swagger.yaml

```
@@ -331,6 +331,42 @@ paths:
         "500":
           $ref: "#/components/responses/internal_server_error"
 
+  /blocks/slot/{slot_number}:
```

```suggestion
  /blocks/slots/{slot_number}:
```
We should probably stick to plurals in this version.